custom/lib_portal/data/lib_data.xml

```diff
@@ -0,0 +1,164 @@
+            <field name="languages_known" eval="[(6, 0, [ref('base.lang_en'), ref('base.lang_ar'), ref('base.lang_bn_IN')])]"/>
+            <field name="image_1920" type="base64" file="lib_portal/static/img/res_partner_author_2.jpeg"/>
+        </record>
+
+        <record id="partner_authors_3" model="res.partner">
+            <field name="company_type">person</field>
+            <field name="is_author">True</field>
+            <field name="name">Jairam Ramesh</field>
+            <field name="street">Currency Street 4</field>
+            <field name="city">Columbus</field>
+            <field name="state_id" ref='base.state_us_30'/>
+            <field name="zip">110021</field>
+            <field name="country_id" ref='base.us'/>
+            <field name="website">www.jaiwritings.com</field>
+            <field name="email">jairamramesh@example.net</field>
+            <field name="languages_known" eval="[(6, 0, [ref('base.lang_en'), ref('base.lang_ar'), ref('base.lang_bn_IN')])]"/>
+            <field name="image_1920" type="base64" file="lib_portal/static/img/res_partner_author_3.jpeg"/>
+        </record>
+
+        <!-- Initial data for interests -->
+
+        <record id="res_partner_interest_0" model="res.partner.category">
+            <field name="name">Thriller</field>
+        </record>
+
+        <record id="res_partner_interest_1" model="res.partner.category">
+            <field name="name">Scifi</field>
+        </record>
+
+        <record id="res_partner_interest_2" model="res.partner.category">
+            <field name="name">Mystery</field>
+        </record>
+
+        <record id="res_partner_interest_3" model="res.partner.category">
+            <field name="name">Romantic</field>
+        </record>
+        
+        <record id="res_partner_interest_4" model="res.partner.category">
+            <field name="name">Fantasy</field>
+        </record>
+
+        <record id="res_partner_interest_5" model="res.partner.category">
+            <field name="name">Technical</field>
+        </record>
+
+        <record id="res_partner_interest_6" model="res.partner.category">
+            <field name="name">Biography</field>
+        </record>
+
+            <!-- Initial data for product category -->
+
+        <record id="product_category_1" model="product.category">
+            <field name="name">Biography</field>
+        </record>
+
+        <record id="product_category_2" model="product.category">
+            <field name="name">Fanstasy</field>
+        </record>
+
+         <record id="product_category_3" model="product.category">
+            <field name="name">Scifi</field>
+        </record>
+
+         <record id="product_category_4" model="product.category">
+            <field name="name">Technical</field>
+        </record>
+
+        <record id="product_category_5" model="product.category">
+            <field name="name">Mystery</field>
+        </record>
+        
```

Comment:
The interests and the genres are basically the same things, so why not using just one model for that?

Reply:
yeah..I am reusing the existing fields in the models.So,may be I will use the ref attribute instead of writing the same values again.Please correct me if I am wrong